src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -240,4 +241,11 @@
 	 * @since 5.1
 	 */
 	boolean storeEmptyValue() default true;
+	
+	/**
+	 * overrides the mapping field type which otherwise will be taken from corresponding {@link FieldType}
+	 *
+	 * @since 5.4
+	 */
+	String mappedName() default "";
```

Comment:
```suggestion
	String mappedTypeName() default "";
```
to not confuse with the field name